pyproject.toml

```diff
@@ -1,13 +1,29 @@
-[project]
+[tool.poetry]
 name = "octreelib"
-dynamic = ["dependencies"]
-version = "0.0.5"
+version = "0.0.0"
+description = "Library for representing point clouds as OcTrees in SLAM"
+authors = [
+  "Kiselyov Mikhail <email@email.email>",
```

Comment:
I don't know your email to submit it here

Reply:
kiselev.0353@gmail.com